MacrobenchmarkSample/build.gradle

```
@@ -17,29 +17,30 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        agpVersion = '7.3.0'
+        agpVersion = "7.3.1"
         targetSdkVersion = 33
         minSdkVersion = 21
         compileSdkVersion = 33
 
-        activityVersion='1.6.1'
-        appcompatVersion='1.5.1'
-        benchmarkVersion = "1.1.0"
+        activityVersion="1.7.0-alpha02"
+        appcompatVersion="1.5.1"
+        benchmarkVersion = "1.1.1"
         composeCompilerVersion = "1.3.1"
         composeUiVersion = "1.3.0"
-        constraintLayoutVersion='2.1.4'
-        coreVersion='1.9.0'
+        constraintLayoutVersion="2.1.4"
+        coreVersion="1.9.0"
         coroutinesVersion = "1.6.4"
-        curtainsVersion = '1.2.4'
-        espressoCoreVersion = '3.4.0'
-        jUnitVersion = '4.13.2'
+        curtainsVersion = "1.2.4"
+        dataStoreVersion = "1.0.0"
+        espressoCoreVersion = "3.4.0"
+        jUnitVersion = "4.13.2"
         kotlinVersion = "1.7.10"
-        lifecycleVersion = '2.5.1'
-        materialVersion='1.7.0'
-        profileInstallerVersion = '1.2.0'
-        testExtVersion = '1.1.3'
-        tracingVersion = '1.1.0'
-        uiAutomatorVersion = '2.2.0'
+        lifecycleVersion = "2.5.1"
+        materialVersion="1.7.0"
+        profileInstallerVersion = "1.2.0"
```

Let's use alpha for now so that we enable BLP on API 32+.
(Once 1.2.1 is released, we can downgrade to stable)

```suggestion
        profileInstallerVersion = "1.3.0-alpha02"
```

---

Nope, let's keep things stable for now.